
code: ```
import rsa
import os
import base64

for i in range(20):
    print(i + 1)
    public, private = rsa.newkeys(10240, poolsize=os.cpu_count()-1)
    open(f'public{i}.txt', 'w').write(base64.b64encode(public.save_pkcs1()).decode())
    open(f'private{i}.txt', 'w').write(base64.b64encode(private.save_pkcs1()).decode())```

traceback: ```1
1
1
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 125, in _main
    prepare(preparation_data)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 236, in prepare
    _fixup_main_from_path(data['init_main_from_path'])
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 287, in _fixup_main_from_path
    main_content = runpy.run_path(main_path,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 289, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 96, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Joseph\Desktop\python\rsa_stuff\keys\generate.py", line 8, in <module>
    public, private = rsa.newkeys(10240, poolsize=os.cpu_count()-1)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 834, in newkeys
    (p, q, e, d) = gen_keys(nbits, getprime_func, accurate=accurate, exponent=exponent)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 776, in gen_keys
    (p, q) = find_p_q(nbits // 2, getprime_func, accurate)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 670, in find_p_q
    p = getprime_func(pbits)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 828, in getprime_func
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 125, in _main
    prepare(preparation_data)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 236, in prepare
    _fixup_main_from_path(data['init_main_from_path'])
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 287, in _fixup_main_from_path
    main_content = runpy.run_path(main_path,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 289, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 96, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Joseph\Desktop\python\rsa_stuff\keys\generate.py", line 8, in <module>
    public, private = rsa.newkeys(10240, poolsize=os.cpu_count()-1)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 834, in newkeys
    (p, q, e, d) = gen_keys(nbits, getprime_func, accurate=accurate, exponent=exponent)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 776, in gen_keys
    (p, q) = find_p_q(nbits // 2, getprime_func, accurate)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 670, in find_p_q
    p = getprime_func(pbits)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 828, in getprime_func
    return parallel.getprime(nbits, poolsize=poolsize)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\parallel.py", line 68, in getprime
    p.start()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\context.py", line 336, in _Popen
    return Popen(process_obj)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\popen_spawn_win32.py", line 45, in __init__
    prep_data = spawn.get_preparation_data(process_obj._name)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 154, in get_preparation_data
    _check_not_importing_main()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 134, in _check_not_importing_main
    raise RuntimeError('''
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
    return parallel.getprime(nbits, poolsize=poolsize)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\parallel.py", line 68, in getprime
    p.start()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\context.py", line 336, in _Popen
    return Popen(process_obj)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\popen_spawn_win32.py", line 45, in __init__
    prep_data = spawn.get_preparation_data(process_obj._name)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 154, in get_preparation_data
    _check_not_importing_main()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 134, in _check_not_importing_main
    raise RuntimeError('''
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
1
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 125, in _main
    prepare(preparation_data)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 236, in prepare
    _fixup_main_from_path(data['init_main_from_path'])
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 287, in _fixup_main_from_path
    main_content = runpy.run_path(main_path,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 289, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 96, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "C:\Users\Joseph\Desktop\python\rsa_stuff\keys\generate.py", line 8, in <module>
    public, private = rsa.newkeys(10240, poolsize=os.cpu_count()-1)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 834, in newkeys
    (p, q, e, d) = gen_keys(nbits, getprime_func, accurate=accurate, exponent=exponent)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 776, in gen_keys
    (p, q) = find_p_q(nbits // 2, getprime_func, accurate)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 670, in find_p_q
    p = getprime_func(pbits)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 828, in getprime_func
    return parallel.getprime(nbits, poolsize=poolsize)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\parallel.py", line 68, in getprime
    p.start()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 121, in start
    self._popen = self._Popen(self)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\context.py", line 336, in _Popen
    return Popen(process_obj)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\popen_spawn_win32.py", line 45, in __init__
    prep_data = spawn.get_preparation_data(process_obj._name)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 154, in get_preparation_data
    _check_not_importing_main()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\spawn.py", line 134, in _check_not_importing_main
    raise RuntimeError('''
RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
Traceback (most recent call last):
  File "C:\Users\Joseph\Desktop\python\rsa_stuff\keys\generate.py", line 8, in <module>
    public, private = rsa.newkeys(10240, poolsize=os.cpu_count()-1)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 834, in newkeys
    (p, q, e, d) = gen_keys(nbits, getprime_func, accurate=accurate, exponent=exponent)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 776, in gen_keys
    (p, q) = find_p_q(nbits // 2, getprime_func, accurate)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 670, in find_p_q
    p = getprime_func(pbits)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\key.py", line 828, in getprime_func
    return parallel.getprime(nbits, poolsize=poolsize)
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\site-packages\rsa\parallel.py", line 70, in getprime
    result = pipe_recv.recv()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\connection.py", line 255, in recv
    buf = self._recv_bytes()
  File "C:\Users\Joseph\AppData\Local\Programs\Python\Python310\lib\multiprocessing\connection.py", line 310, in _recv_bytes
    waitres = _winapi.WaitForMultipleObjects(
KeyboardInterrupt

Process finished with exit code -1073741510 (0xC000013A: interrupted by Ctrl+C)```